iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/ActivePowerControlAdder.java

```diff
@@ -22,4 +22,10 @@ default Class<ActivePowerControl> getExtensionClass() {
     ActivePowerControlAdder<I> withDroop(double droop);
 
     ActivePowerControlAdder<I> withParticipationFactor(double participationFactor);
+
+    boolean getParticipate();
+
+    double getDroop();
+
+    double getParticipationFactor();
```

Comment:
to remove, you did this because of the incorrect g.getExtension(ActivePowerControl**Adder**.class) above